### ISSUE TITLE
Ghostbusters: The Video Game Remastered - CUSA15493

### ISSUE THREAD

**Issue by Ordinary205** [red-prig/fpps4-game-compatibility#1668]
### Game Name

Ghostbusters: The Video Game Remastered

### Game code

CUSA15493

### Game version

1.00

### Used emulator's version

d6eef26

### Current status

Boots

### Error

An unhandled exception occurred at $0000000800842FB1:
EAssertionFailed: SCE_AUDIO_OUT_PARAM_FORMAT_S16_8CH (ps4_libsceaudioout.pas, line 671)
  $0000000800842FB1

An unhandled exception occurred at $000000010009DD7F:
EAccessViolation: Access violation
  $000000010009DD7F
  $00000001000746CF
  $0000000100011E61
  $0000000100012122
  $000000010001214A
  $0000000100012170
  $000000003781E6A0
  $000000003781FD70
  $0000000000000022
  $000000003781F580
  $000000000010000F
  $00000001000199D9
  $00007FFB46AD242F
  $00007FFB46A60939
  $000000010001979A
  $00007FFB46AD23AF
  $00007FFB46A814B4

### Description

Crashes after this screen:
![Ghost busters 1](https://github.com/red-prig/fpps4-game-compatibility/assets/125112932/c39f749c-2f8b-43c5-ad0c-42477bbb3d93)

### Screenshots

![ghost busters](https://github.com/red-prig/fpps4-game-compatibility/assets/125112932/5729136a-1592-454f-a549-440930ff3eae)

![Interesting error from FPPS4](https://github.com/red-prig/fpps4-game-compatibility/assets/125112932/46f04411-5ebe-466a-abd7-38eb4e6372fe)

**Comment by GAMERRPCS3** [comment]
try that.

**Comment by dev-channe** [comment]
try that.--libSceAudioOut

**Comment by Ordinary205** [comment]
I tried doing that but it didn't work.
![Error](https://github.com/red-prig/fpps4-game-compatibility/assets/125112932/8d6e9faf-1fc1-4eb9-8723-620518374483)

**Comment by dev-channe** [comment]
> I tried doing that but it didn't work. ![Error](https://user-images.githubusercontent.com/125112932/254969584-8d6e9faf-1fc1-4eb9-8723-620518374483.PNG)

so really I don't know if it's due to my GPU.

**Comment by Ordinary205** [comment]
Maybe I need to try testing the other game ID such as CUSA15558 to see if that makes a difference, but I'll do that later.

**Comment by dev-channe** [comment]
> Maybe I need to try testing the other game ID such as CUSA15558 to see if that makes a difference, but I'll do that later.

if there's one thing I'm sure of. for this game I didn't delete any files, I tested them out of curiosity. after more than 6 months since the last test.

**Comment by Ordinary205** [comment]
Yeah, I'm not really sure whats going on with this issue, and I dont remember if I changed something when testing this.
CUSA15558:
![1](https://github.com/red-prig/fpps4-game-compatibility/assets/125112932/2b507313-c12e-4b00-9f65-afcd99b3137b)

CUSA15493:
![Error](https://github.com/red-prig/fpps4-game-compatibility/assets/125112932/63cf0a81-9fd1-4d4c-86c7-0651c8c3372f)

**Comment by gandalfthewhite19890404** [comment]
Upload whole log with test of 15493.
And do not report here anything about 15558, there is a separate issue.

**Comment by Ordinary205** [comment]
> Upload whole log with test of 15493. And do not report here anything about 15558, there is a separate issue.

My bad, but here it is.
[Log.txt](https://github.com/red-prig/fpps4-game-compatibility/files/12117030/Log.txt)

**Comment by Ordinary205** [comment]
The game can now reach menus with DISABLE_FMW_HACK enabled.

![Ghostbusters 1](https://github.com/red-prig/fpps4-game-compatibility/assets/125112932/1c09ae79-22ee-4add-b4f7-5a70e8f446b7)

![Ghostbusters 2](https://github.com/red-prig/fpps4-game-compatibility/assets/125112932/4f4027ca-dcaf-4f66-882b-7087c5abab0e)

This is an error when trying to reach ingame.
![Ghostbusters 3](https://github.com/red-prig/fpps4-game-compatibility/assets/125112932/beafabf2-2792-4cc5-b5cd-b12368b9b507)

Status: menus

**Comment by GamingDucking** [comment]
Retest

**Comment by Ordinary205** [comment]
Now the game goes further during menus, but sadly the game crashes after the cutscene has ended, and using DISABLE_FMW_HACK is no longer mandatory.
![Ghostbusters](https://github.com/red-prig/fpps4-game-compatibility/assets/125112932/8b48efba-4750-412e-afdc-708beb65a09f)

![Ghostbusters 2](https://github.com/red-prig/fpps4-game-compatibility/assets/125112932/f442754d-8ba7-4081-b58a-e09f3e4b6fe2)

![Ghostbusters 3](https://github.com/red-prig/fpps4-game-compatibility/assets/125112932/9dab520d-3d02-420b-a260-6079459c658f)

![Ghostbusters 4](https://github.com/red-prig/fpps4-game-compatibility/assets/125112932/d0631bf6-2f52-4d12-9aa6-ffcd2321f05e)

Also the audio behaves louder during cutscenes.

**Comment by Ordinary205** [comment]
Please remove "hacks-needed", I managed to reach menus without having DISABLE_FMW_HACK.
